stream-chat-android-ui-components/src/main/res/values/attrs_message_list_view.xml

```diff
@@ -338,6 +359,15 @@
             <flag name="bold" value="1" />
             <flag name="italic" value="2" />
         </attr>
-
+        <attr name="streamUiMessageListProgressBarStyle" format="reference" />
```

Comment:
Maybe instead of style, we could give the possibility to provide a custom progress bar layout? We are doing that in `ChannelListView` [here](https://github.com/GetStream/stream-chat-android/blob/develop/stream-chat-android-ui-components/src/main/res/values/attrs_channel_list_view.xml#L67). What do you think about this?